website/views/pages/pricing.ejs

```diff
@@ -1,3 +1,1277 @@
+                      />
+                      <p class="ml-2 my-0">Osquery agent autoupdates†</p>
+                    </div>
+                    <div class="col-3 col-md-2">
+                      <img
+                        alt="a check mark"
+                        src="/images/check-16x16@2x.png"
+                        style="width: 16px; height: 16px"
+                      />
+                    </div>
+                    <div class="col-3 col-md-2">
+                      <img
+                        alt="a check mark"
+                        src="/images/check-16x16@2x.png"
+                        style="width: 16px; height: 16px"
+                      />
+                    </div>
+                  </div>
+                  <div class="row mb-3">
+                    <div class="col-6 col-md-8 d-flex">
+                      <img
+                        alt="a bullet point ellipse"
+                        src="/images/ellipse-6x8@2x.png"
+                        class="mt-2"
+                        style="width: 6.4px; height: 8px"
+                      />
+                      <p class="ml-2 my-0">Self-hosted agent autoupdates†</p>
+                    </div>
+                    <div class="col-3 col-md-2"></div>
+                    <div class="col-3 col-md-2">
+                      <img
+                        alt="a check mark"
+                        src="/images/check-16x16@2x.png"
+                        style="width: 16px; height: 16px"
+                      />
+                    </div>
+                  </div>
+                  <div class="row mb-3">
+                    <div class="col-6 col-md-8 d-flex">
+                      <img
+                        alt="a bullet point ellipse"
+                        src="/images/ellipse-6x8@2x.png"
+                        class="mt-2"
+                        style="width: 6.4px; height: 8px"
+                      />
+                      <p class="ml-2 my-0">
+                        Mass deploy custom osquery extensions*
+                      </p>
+                    </div>
+                    <div class="col-3 col-md-2"></div>
+                    <div class="col-3 col-md-2">
+                      <img
+                        alt="a check mark"
+                        src="/images/check-16x16@2x.png"
+                        style="width: 16px; height: 16px"
+                      />
+                    </div>
+                  </div>
+                  <p style="font-size: 13px; left: 0" class="mt-5 mb-0 w-100">
+                    * Coming soon
+                  </p>
+                  <p style="font-size: 13px; left: 0" class="mb-0 w-100">
+                    † Customer promise
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <% /*
+  <div style="max-width: 800px; color: #fff" class="py-5 mx-auto">
```

Comment:
can we delete these comments? they will there in history if we need later.

Reply:
I copy+paste the commented-out existing FAQ section from the /homepage.ejs. The Figma /pricing has a FAQ section drawn up.

Reply:
The FAQ section won't be a part of these changes. I think we can remove the comment section.